R-package/tests/testthat/test_callbacks.R

```diff
@@ -184,15 +184,19 @@ test_that("cb.save.model works as expected", {
   expect_equal(xgb.ntree(b1), 1)
   b2 <- xgb.load('xgboost_02.model')
   expect_equal(xgb.ntree(b2), 2)
+
+  xgb.config(b2) <- xgb.config(bst)
+  expect_equal(xgb.config(bst), xgb.config(b2))
```

Comment:
We should also test `xgb.serialize` and `xgb.unserialize`, and see whether they preserve configuration.

Reply:
Added test.